JSDemos/Demos/Charts/Stock/description.md

```diff
@@ -1 +1,10 @@
-_Stock_ series help display variations in&nbsp;stock prices over the course of&nbsp;a&nbsp;day. High and low prices are represented by&nbsp;the bottom and top values of&nbsp;the vertical line and open and close prices are represented by&nbsp;left and right tick marks.
\ No newline at end of file
+A stock [series type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. You can specify the stock series type for each object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[stock](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/StockSeries/) object to configure properties of the Chart stock series.
+
+In stock series, high and low prices are represented by the bottom and top values of the vertical line. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for high and low prices respectively.
+
+Open and close prices are represented by left and right tick marks. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties.
+
+To mirror the changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To specify whether high, low, open or close prices of points the component should compare, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
```

Comment:
The words like "whether" or "if" have binary logic, basically, you can answer such a question only with "yes" or "no". However, you list four items after "whether", which is definitely incorrect.

For example, we can rewrite this paragraph as follows:

```suggestion
To reflect changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To switch between high, low, open or close prices of points as sources for data comparison, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
```